components/zoom_admin/zoom_admin.app.mjs

```diff
@@ -341,5 +341,15 @@ export default {
       });
       return get(res, "data");
     },
+    async listWebinarParticipants(webinarID, pageSize, nextPageToken) {
+      const { data } = await this._makeRequest({
```

Comment:
What do you think of keeping this method in the component's file, since it's being created just for that component?

The only change required, besides moving it there, would be
```suggestion
      const { data } = await this.zoomAdmin._makeRequest({
```

And so no version bumps would be required at all for the other components, as the app file would no longer need to be changed.